x-pack/test_serverless/shared/services/index.ts

```diff
@@ -25,5 +26,7 @@ export const services = {
   svlCommonApi: SvlCommonApiServiceProvider,
   svlReportingApi: SvlReportingServiceProvider,
   svlUserManager: commonFunctionalServices.samlAuth,
+  samlAuth: commonFunctionalServices.samlAuth, // <--temp workaround until we can unify naming
```

Comment:
@elastic/appex-qa It looks like `svlUserManager` is referenced all other the place, but `samlAuth` is required by `RoleScopedSupertestProvider`. Instead of resolving this here (it will require review from many teams), can we do this in a follow up?

Reply:
I believe that's what we agreed on with @dmlemeshko :) Keeping both `svlUserManager` and `samlAuth` for now, as unifying them would require quite a bit of work.

Reply:
naming is hard, wish I call it `samlAuth` originally. Hopefully after some tests refactoring (including deployment agnostic migration) we can renaming service in all the test files. But not now :)